azure-toolkit-libs/azure-toolkit-keyvault-lib/src/main/java/com/microsoft/azure/toolkit/lib/keyvault/KeyVault.java

```diff
@@ -147,5 +151,19 @@ public SecretAsyncClient getSecretClient() {
     public KeyAsyncClient getKeyClient() {
         return Optional.ofNullable(getRemote()).map(Vault::keyClient).orElse(null);
     }
+
+    @Nullable
+    public static Action<String> getAccessPolicyConfiureAction(@Nonnull KeyVault vault) {
+        final String url = vault.getPortalUrl() + "/access_configuration";
```

Comment:
Concatinating strings to construct a URL is not the right way. Please use URL class/functions instead.